DotNetRu.Android/DotNetRu.Android.csproj

```diff
@@ -35,7 +35,7 @@
     <RunCodeAnalysis>false</RunCodeAnalysis>
     <AssemblyName>DotNetRu.Droid</AssemblyName>
     <EmbedAssembliesIntoApk>false</EmbedAssembliesIntoApk>
-    <AndroidSupportedAbis>armeabi-v7a</AndroidSupportedAbis>
+    <AndroidSupportedAbis>armeabi-v7a;x86</AndroidSupportedAbis>
```

Comment:
Please create a new configuration for Emulator if you need to.
Debug builds for real device should be fast, we don't want to spent time for compiling for unneeded architecture

Reply:
@pfedotovsky fixed